metadata-ingestion/setup.py

```diff
@@ -505,6 +506,7 @@ def get_long_description():
             "nifi",
             "vertica",
             "mode",
+            "fivetran",
```

Comment:
why is this needed?

Reply:
Now we added one unit test case on fivetran source